Makefile is not creating the destination directory when running the copy recipe.

If the destination directory does not exist (ex. `˜/.bin`) the `make copy` command doesn't create the destination directory by default giving the following error:

```
cp ./terraform-lsp ~/.bin/ && cp ./terraform-lsp ~/
cp: directory ˜/.bin does not exist
make: *** [copy] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -6,8 +6,11 @@ DST     ?= ~/.bin/
 terraform-lsp:
 	go build -ldflags "-X main.GitCommit=$(COMMIT) -X main.Version=$(VERSION) -X main.Date=$(DATE)"
 
-copy: terraform-lsp
+copy: terraform-lsp | create-dir
 	cp ./terraform-lsp $(DST) && cp ./terraform-lsp ~/
+
+create-dir:
+	mkdir -p $(DST)
 
 clean:
 	rm -f terraform-lsp

